bin/openapi3/python-experimental-petstore.sh

```diff
@@ -32,3 +32,8 @@ yaml="modules/openapi-generator/src/test/resources/3_0/python-experimental/petst
 ags="generate -t modules/openapi-generator/src/main/resources/python -i $yaml -g python-experimental -o samples/openapi3/client/petstore/python-experimental/ --additional-properties packageName=petstore_api $@"
 
 java $JAVA_OPTS -jar $executable $ags
+
+yaml="modules/openapi-generator/src/test/resources/3_0/extensions/x-auth-id-alias.yaml"
+ags="generate -t modules/openapi-generator/src/main/resources/python -i $yaml -g python-experimental -o samples/openapi3/client/extensions/x-auth-id-alias/python-experimental/ --additional-properties packageName=x_auth_id_alias $@"
```

Comment:
If we decide to keep this separate sample can you put in a python-experimental folder so our uses see both python-experimental samples at the same place?
Something like
- `samples/openapi3/client/petstore/python-experimental/general_sample`
- `samples/openapi3/client/petstore/python-experimental/auith_id_alias_sample`

Also, how about [this solution](https://github.com/OpenAPITools/openapi-generator/pull/6469#discussion_r435337114) of separating our existing spec into a python-experimental only version and a java version. That will let us add this feature in a python-experimental only spec, have no impact on other generators, not impact live server testing, and will keep it clear to users what sample to look at.

Reply:
I would like to generate other clients for this feature. I don't think it's `python-experimental` specific.

Reply:
Ah, I understand. That's fine then